src/Chat.tsx

```diff
@@ -74,6 +75,13 @@ export class Chat extends React.Component<ChatProps, {}> {
 
         konsole.log("BotChat.Chat props", props);
 
+        if (props.pastHistory) {
+            this.store.dispatch<HistoryAction>({
```

Comment:
we've been switching to using `ChatActions` (which includes `HistoryAction`), since a given action might actually be handled by multiple reducers. One day I might get rid of the individual `{reducer}Action` types for this reason.